config/settings.js

```diff
@@ -0,0 +1,5 @@
+require('dotenv').config();
+
+exports.showUserOptIn = () => {
+  return process.env.SHOW_USER_OPT_IN || false;
```

Comment:
Kunnen we dit verplaatsen van de env file naar de database? In de clients.config zou perfect zijn. Ik kan me situaties voorstellen waar we dit niet voor elke site aan willen zetten.